web_src/less/_markup.less

```diff
@@ -538,7 +538,7 @@
   }
 }
 
-.markdown-block-error {
+.markup-block-error {
```

Comment:
This also needs to be changed in `web_src/js/markdown/mermaid.js`

Reply:
Fixed in 82936c5